ui/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -267,6 +290,29 @@ export default class PermissionConnect extends Component {
                 />
               )}
             />
+            {
+              ///: BEGIN:ONLY_INCLUDE_IN(flask)
+            }
+            <Route
+              path={snapInstallPath}
+              exact
+              render={() => (
+                <SnapInstall
+                  request={permissionsRequest || {}}
```

Comment:
`permissionsRequest` should not be empty :thinking: At least not unless `redirecting` is `true`, in which case this would not be rendered. See the `componentDidUpdate` lifecycle function

Reply:
Check PermissionPageContainer route, the same safeguard is there in place.